sdk/instruction/src/account_meta.rs

```diff
@@ -0,0 +1,105 @@
+#[cfg(any(feature = "std", target_arch = "wasm32"))]
```

Comment:
Yeah I took another look and realised this can be much simpler, had been assuming the wasm32 code needed std because we have other places where that's the case but it's not the case here